src/scss/_responsive.scss

```diff
@@ -0,0 +1,23 @@
+// For more this naming system:
+// http://alexbea.com/blog/ergonomic-breakpoint-variables
```

Comment:
this mf

Reply:
lol You got me all hyped. I'd also be fine removing. I was thinking of people opening issues later questioning.